### PR TITLE
Section added to README.md giving a high level of Circus Train extens…

### PR DESCRIPTION
**Pull request by saktheeshb**
Section added to README.md giving a high level of Circus Train extensions that are publicly available, namely

https://github.com/ExpediaInceCommercePlatform/circus-train-datasqueeze
https://github.com/HotelsDotCom/circus-train-bigquery

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -896,6 +896,16 @@ Note that the Circus Train configuration only supports one set of AWS credential
           table-name: aws-a_table
           table-location: s3://<aws-a-data-bucket>/<pata-to-aws-a_db>/aws-a_table
 
+## Circus Train extensions
```
Comment (massdosage) [inline README.md]: Could you move this instead to be the final sub-section under "Extension Points" as I think it fits in more logically there. So I'd suggest a `###` level sub-heading and we should call it "External Extensions" instead to be clear that these are outside of Circus Train itself.

File: README.md [hunk 2]
```diff
@@ -896,6 +896,16 @@ Note that the Circus Train configuration only supports one set of AWS credential
...
+
+### Circus Train DataSqueeze Copier
```
Comment (massdosage) [inline README.md]: This and the next would move to level `####`

File: README.md [hunk 3]
```diff
@@ -896,6 +896,16 @@ Note that the Circus Train configuration only supports one set of AWS credential
...
+This extension implements new Copiers for Circus Train which integrate DataSqueeze.
```
Comment (massdosage) [inline README.md]: Could you reword this as "The DataSqueeze extension provides new Copiers for Circus Train which compact the data being replicated in various ways" and then make the word "DataSqueeze" a hyperlink to the URL instead of listing it in plain text.

File: README.md [hunk 4]
```diff
@@ -896,6 +896,16 @@ Note that the Circus Train configuration only supports one set of AWS credential
...
+url: https://github.com/ExpediaInceCommercePlatform/circus-train-datasqueeze
+
+### circus-train-bigquery
+This Circus Train plugin enables the conversion of BigQuery tables to Hive.
```
Comment (massdosage) [inline README.md]: Similar here - make this more like "The Circus Train BigQuery extension enables the conversion of Google BigQuery tables to Hive" and make the text `Circus Train BigQuery` a hyperlink to the URL.

File: README.md (hunk 1 above)
Comment (saktheeshb) [reply README.md]: Thanks for your comment. It has been added now.

File: README.md [hunk 5]
```diff
@@ -672,6 +672,18 @@ Circus Train can be extended in various ways. This is an advanced feature that t
 * `LocationManagers` look after the source and replica locations and are an ideal point to implement both snapshot isolation and retired data clean-up.
 * `CompositeCopierFactory` allows the provision of multiple copiers for the same table in order to add functionality to the copy process, e.g. introduce compaction, copy to multiple destinations, etc.
 
+### External Extensions
+
+#### Circus Train DataSqueeze Copier
+This extension implements new Copiers for Circus Train which integrate DataSqueeze.
```
Comment (massdosage) [inline README.md]: Either make this:
```
This extension implements new Copiers for Circus Train which integrate DataSqueeze:
```
note the `:`.

*Or* make the word `DataSqueeze` linkable like this and remove the separate URL:
```
This extension implements new Copiers for Circus Train which integrate [DataSqueeze](https://github.com/ExpediaInceCommercePlatform/circus-train-datasqueeze).
```

```

File: README.md (hunk 2 above)
Comment (saktheeshb) [reply README.md]: Done.

File: README.md [hunk 6]
```diff
@@ -672,6 +672,18 @@ Circus Train can be extended in various ways. This is an advanced feature that t
...
+
+https://github.com/ExpediaInceCommercePlatform/circus-train-datasqueeze
+
+#### circus-train-bigquery
+This Circus Train plugin enables the conversion of BigQuery tables to Hive.
```
Comment (massdosage) [inline README.md]: Same here as above.

File: README.md (hunk 3 above)
Comment (saktheeshb) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (saktheeshb) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (saktheeshb) [reply README.md]: could you please check this again,  I have added that.

File: README.md [hunk 7]
```diff
@@ -672,6 +672,14 @@ Circus Train can be extended in various ways. This is an advanced feature that t
 * `LocationManagers` look after the source and replica locations and are an ideal point to implement both snapshot isolation and retired data clean-up.
 * `CompositeCopierFactory` allows the provision of multiple copiers for the same table in order to add functionality to the copy process, e.g. introduce compaction, copy to multiple destinations, etc.
 
+### External Extensions
+
+#### Circus Train DataSqueeze Copier
+The [DataSqueeze](https://github.com/ExpediaInceCommercePlatform/circus-train-datasqueeze) extension provides new Copiers for Circus Train which compact the data being replicated in various ways
```
Comment (massdosage) [inline README.md]: Looks good, last change request (I promise!) - put a full stop at the end of both sentences ;)

Comment (saktheeshb) [reply README.md]: Sure :-). It's done

File: README.md [hunk 8]
```diff
@@ -672,6 +672,14 @@ Circus Train can be extended in various ways. This is an advanced feature that t
 * `LocationManagers` look after the source and replica locations and are an ideal point to implement both snapshot isolation and retired data clean-up.
 * `CompositeCopierFactory` allows the provision of multiple copiers for the same table in order to add functionality to the copy process, e.g. introduce compaction, copy to multiple destinations, etc.
 
+### External Extensions
+
+#### Circus Train DataSqueeze Copier
+The [DataSqueeze](https://github.com/ExpediaInceCommercePlatform/circus-train-datasqueeze) extension provides new Copiers for Circus Train which compact the data being replicated in various ways.
+
+#### circus-train-bigquery
```
Comment (AnanaMJ) [inline README.md]: I think this can be "Circus Train BigQuery" since the one above isn't "circus-train-datasqueeze" :)

Comment (massdosage) [reply README.md]: Agreed, I thought the same thing earlier, I'll fix that quickly now.

**Comment by massdosage** [comment]
Fixes #117.

Thanks for the contribution @saktheeshb
